packages/spark-core/utilities/_utilities.scss

```diff
@@ -2,166 +2,301 @@
+}
+
+.sprk-u-Color--red-light {
+  color: $red-light !important;
+}
+
+.sprk-u-Color--red-medium {
+  color: $red-medium !important;
+}
+
+.sprk-u-Color--red-dark {
+  color: $red-dark !important;
+}
+
+.sprk-u-Color--orange-medium {
+  color: $orange-medium !important;
+}
+
+.sprk-u-Color--orange-dark {
+  color: $orange-dark !important;
+}
+
+.sprk-u-Color--gray-light {
+  color: $gray-light !important;
+}
+
+.sprk-u-Color--gray-medium {
+  color: $gray-medium !important;
+}
+
+.sprk-u-Color--gray-dark {
+  color: $gray-dark !important;
+}
+
+.sprk-u-Color--black {
+  color: $black !important;
+}
+
+.sprk-u-Color--white {
+  color: $white !important;
+}
+
+// Background color
 .sprk-u-BackgroundColor--green-light {
-  background-color: $green-light;
+  background-color: $green-light !important;
 }
 
 .sprk-u-BackgroundColor--green-medium {
-  background-color: $green-medium;
+  background-color: $green-medium !important;
 }
 
 .sprk-u-BackgroundColor--green-dark {
-  background-color: $green-dark;
+  background-color: $green-dark !important;
 }
 
 .sprk-u-BackgroundColor--blue-light {
-  background-color: $blue-light;
+  background-color: $blue-light !important;
 }
 
 .sprk-u-BackgroundColor--blue-medium {
-  background-color: $blue-medium;
+  background-color: $blue-medium !important;
 }
 
 .sprk-u-BackgroundColor--blue-dark {
-  background-color: $blue-dark;
+  background-color: $blue-dark !important;
+}
+
+.sprk-u-BackgroundColor--purple-medium {
+  background-color: $purple-medium !important;
 }
 
-.sprk-u-BackgroundColor--purple {
-  background-color: $purple-medium;
+.sprk-u-BackgroundColor--purple-dark {
+  background-color: $purple-dark !important;
 }
 
 .sprk-u-BackgroundColor--red-light {
-  background-color: $red-light;
+  background-color: $red-light !important;
 }
 
 .sprk-u-BackgroundColor--red-medium {
-  background-color: $red-medium;
+  background-color: $red-medium !important;
 }
 
 .sprk-u-BackgroundColor--red-dark {
-  background-color: $red-dark;
+  background-color: $red-dark !important;
 }
 
 .sprk-u-BackgroundColor--orange-medium {
-  background-color: $orange-medium;
+  background-color: $orange-medium !important;
 }
 
 .sprk-u-BackgroundColor--orange-dark {
-  background-color: $orange-dark;
+  background-color: $orange-dark !important;
 }
 
 .sprk-u-BackgroundColor--gray-light {
-  background-color: $gray-light;
+  background-color: $gray-light !important;
 }
 
 .sprk-u-BackgroundColor--gray-medium {
-  background-color: $gray-medium;
+  background-color: $gray-medium !important;
 }
 
 .sprk-u-BackgroundColor--gray-dark {
-  background-color: $gray-dark;
+  background-color: $gray-dark !important;
 }
 
 .sprk-u-BackgroundColor--black {
-  background-color: $black;
+  background-color: $black !important;
 }
 
 .sprk-u-BackgroundColor--white {
-  background-color: $white;
+  background-color: $white !important;
+}
+
+//
+// Multi-property utilities
+///
+
+// Accessibility
+.sprk-u-ScreenReaderText {
```

Comment:
I changed this from `VisuallyHidden` because the purpose of this is to make it readable by a screen reader, so I thought this name is more obvious what it's for.